tests/e2e/harness/preload.mjs

```
@@ -0,0 +1,77 @@
+ * write a JSON file describing the LeetCode responses they want, point the
+ * child at it via env, and then drive the server through StdioClientTransport.
+ *
+ * If `E2E_FIXTURE_PATH` is not set, this preload is a no-op apart from
+ * disabling network — useful for lifecycle tests that don't touch LeetCode.
+ */
+import nock from "nock";
+import { readFileSync } from "node:fs";
+
+nock.disableNetConnect();
+
+const fixturePath = process.env.E2E_FIXTURE_PATH;
```

ESLint

[error] 'process' is not defined.

(no-undef)